ROADMAP.md

```diff
@@ -1,35 +1,48 @@
 # Roadmap
 
-## v0.6
-
-### Main Focus
+## Main Focus
 
 The list is sorted by priority
 
-* [x] Request/Response
-* [ ] Zenoh Gateway
-* [ ] Finalize C and C++ API (still some tiny things open)
-* [x] Improve cross-compilation support
+### Planned For v0.7
 
-### Bonus
+* [x] Python Language Bindings
+* [x] Basic Host2Host Communication based on zenoh
+* [ ] `iox2` cli service debugging
+* [ ] Detailed getting started documentation for all supported languages
 
-The list is sorted by priority
+### After v0.7
 
-* [ ] Derive macro for SHM transferable types
-* [ ] proof of concept with <https://github.com/rkyv/rkyv>
-* [ ] `#![no_std]` for low-level crates
-* [ ] Finish ROS 2 rmw binding (requires request/response)
-* [ ] `serde`-based shm serialization to transmit arbitrary types
+* [ ] Embedded Platforms: QNX 7.1 & QNX 8.0
+* [ ] Blackboard Messaging Pattern
 
 ## Backlog
 
+> [!IMPORTANT]
+>
+> **All features in the backlog still require funding.**
```

Comment:
@elfenpiff Do you want to keep this line ? I think it contradicts the tone of the following paragraphs.

Reply:
Oh, overlooked that. I agree with @orecham. I think we want to carry the message that we want to create good software in the first place but for that we also need to buy groceries.

Reply:
I renamed the label to `needs funding`